packages/project/src/v4-specification.ts

```diff
@@ -1136,6 +1145,14 @@ export interface Field {
    */
   id?: string;
 }
+/**
+ * Enum indicating whether data is historical data reported by the grower,
+ * or a projection of future data.
+ */
+enum DataTimeFrame {
+  GrowerReported = 'GROWER_REPORTED',
+  Projected = 'PROJECTED',
```

Comment:
can we just keep the value and key the same?
```suggestion
  GROWER_REPORTED = 'GROWER_REPORTED',
  PROJECTED = 'PROJECTED',
```